File: abc205/c.py
def mypow(x, n):
    res=1
    while n>0:
        if n&1: res*=x
        x*=x
        n>>=1
    return res

def main():
    A,B,C=map(int,input().split())
    x=mypow(A,C)
    y=mypow(B,C)
    print("<" if x<y else ">" if x>y else "=")

if __name__ == "__main__":
    main()

File: abc205/c.c
#include <stdio.h>

long int mypow(long x, int n){
    long int res=1;
    while(n>0){
        if(n&1) res*=x;
        x*=x;
        n>>=1;
    }
    return res;
}

int main(void){
    long int A,B,C,x,y;
    scanf("%ld %ld %ld", &A, &B, &C);
    x=mypow(A,C);
    y=mypow(B,C);
    if (x<y) printf("<\n");
    else if(x>y) printf(">\n");
    else if(x==y) printf("=\n");
}